src/server/main_service.cc

```diff
@@ -68,6 +68,118 @@ std::optional<VarzFunction> engine_varz;
 
 constexpr size_t kMaxThreadSize = 1024;
 
+void UnregisterMonitors(ConnectionContext* server_ctx) {
+  ConnectionState& conn_state = server_ctx->conn_state;
+  if (conn_state.monitor) {
+    // remove monitor on this connection
+    auto token = conn_state.monitor->borrow_token;
+    server_ctx->ChangeMonitor(false /*start*/);
+    token.Wait();
+  }
+}
+
+// The format of the message that are sending is
+// +"time of day" [db-number <lua|unix:path|connection info] "command" "arg1" .. "argM"
+std::string MonitorTimestamp() {
```

Comment:
functions should be verbs. GetMonitorTimestamp, CreateMonitorTimestamp ?

Reply:
ok